src/current/v25.3/alter-table.md

```diff
@@ -1203,71 +1203,87 @@ By default, referenced columns must be in the same database as the referencing f
 
 #### Drop and add a primary key constraint
 
-Suppose that you want to add `name` to the composite primary key of the `users` table, [without creating a secondary index of the existing primary key](#changing-primary-keys-with-add-constraint-primary-key).
+Suppose that you want to add `creation_time` to the composite primary key of the `promo_codes` table, [without creating a secondary index of the existing primary key](#changing-primary-keys-with-add-constraint-primary-key).
 
 {% include_cached copy-clipboard.html %}
 ~~~ sql
-> SHOW CREATE TABLE users;
+SHOW CREATE TABLE promo_codes;
 ~~~
 
 ~~~
-  table_name |                      create_statement
--------------+--------------------------------------------------------------
-  users      | CREATE TABLE users (
-             |     id UUID NOT NULL,
-             |     city VARCHAR NOT NULL,
-             |     name VARCHAR NULL,
-             |     address VARCHAR NULL,
-             |     credit_card VARCHAR NULL,
-             |     CONSTRAINT users_pkey PRIMARY KEY (city ASC, id ASC)
-             | )
+  table_name  |                      create_statement
+--------------+------------------------------------------------------------
+  promo_codes | CREATE TABLE public.promo_codes (
+              |     code VARCHAR NOT NULL,
+              |     description VARCHAR NULL,
+              |     creation_time TIMESTAMP NULL,
+              |     expiration_time TIMESTAMP NULL,
+              |     rules JSONB NULL,
+              |     CONSTRAINT promo_codes_pkey PRIMARY KEY (code ASC)
+              | ) WITH (schema_locked = true);
 (1 row)
 ~~~
 
-1. Add a [`NOT NULL`]({% link {{ page.version.version }}/not-null.md %}) constraint to the `name` column with [`ALTER COLUMN`](#alter-column).
+1. The table is [schema-locked]({% link {{ page.version.version }}/with-storage-parameter.md %}#table-parameters), which means schema changes are disallowed on the table. To make [DDL]({% link {{ page.version.version }}/sql-statements.md %}#data-definition-statements) changes, unlock the table:
+
+	{% include_cached copy-clipboard.html %}
+	~~~ sql
+	ALTER TABLE promo_codes SET (schema_locked = false);
+	~~~
+
+1. Add a [`NOT NULL`]({% link {{ page.version.version }}/not-null.md %}) constraint to the `creation_time` column with [`ALTER COLUMN`](#alter-column):
 
     {% include_cached copy-clipboard.html %}
     ~~~ sql
-    > ALTER TABLE users ALTER COLUMN name SET NOT NULL;
+    ALTER TABLE promo_codes ALTER COLUMN creation_time SET NOT NULL;
     ~~~
 
+1. Turn off autocommit for DDL for the current session, so that multiple DDL statements can execute within one explicit transaction:
```

Comment:
since we are in here touching it pls update to not do multiple DDLs in one transaction, it can lead to sadness